core/runner_actor.py

```diff
@@ -24,14 +25,6 @@ class DMLRunnerActor(DMLRunner):
     """
     DML Runner Actor
 
-    NOTE: This is a wrapper around DMLRunner so that testing on DMLRunner can be 
```

Comment:
I meant to delete the lines with "This class manages the training...This class only executes the jobs." which are now lines 28-35. The lines with the NOTEs were actually useful. Can you bring those lines back and delete 28-35?

Also, yes, even though you copied the method docstrings directly from DMLRunner, I believe we should remove them and just put:

```
"""See method docstring in the original class for more information."""
```

That way we do not maintain documentations in parallel across files.

Reply:
Decided to move away from `ray`, so no need for the actor class anymore. 

Tl;dr resolved.